stories/NestedTransition.js

```diff
@@ -4,24 +4,28 @@ import StoryFixture from './StoryFixture';
 import Fade from './transitions/Fade';
 import Scale from './transitions/Scale';
 
-const FadeAndScale = (props) => (
-  <Fade {...props}>
-    <div>
-      <div>I will fade</div>
-      {/*
-        We also want to scale in at the same time so we pass the `in` state here as well, so it enters
-        at the same time as the Fade.
-
-        Note also the `appear` since the Fade will happen when the item mounts, the Scale transition
-        will mount at the same time as the div we want to scale, so we need to tell it to animate as
-        it _appears_.
-      */}
-      <Scale in={props.in} appear>
-        <div>I should scale</div>
-      </Scale>
-    </div>
-  </Fade>
-)
+const FadeAndScale = (props) => {
+  const fadeRef = React.createRef()
+  const scaleRef = React.createRef()
```

Comment:
`FadeAndScale` is a React function component, so you should use `useRef` instead of `createRef`. I know that it probably doesn't matter here, but I would like to avoid red flags when it's easy.